### PR TITLE
Add documentation to InnerBlocks props that missed it

### PR DESCRIPTION
**Pull request by jorgefilipecosta**
## Description
This PR just documents the InnerBlock properties allowedBlocks, template and templateLock.

### CODE REVIEW

**Review by tofumatt — APPROVED** [review]
Looks good! I made a few tweaks and pushed them–a few things I found a bit unclear so I tried to expand on them/tidy them up.

If you're okay with my changes: 🚢

File: editor/components/inner-blocks/README.md [hunk 1]
```diff
@@ -38,11 +38,72 @@ registerBlockType( 'my-plugin/my-block', {
...
+* **Type:** `String|Boolean`
+
+Template locking of `InnerBlocks` is similar to [Custom Post Type templates locking](https://wordpress.org/gutenberg/handbook/templates/#locking).
+
+Template locking allows locking the `InnerBlocks` area for the current template.
+*Options:*
+
+- `all` — prevents all operations. It is not possible to insert new blocks. Move existing blocks or delete them.
+- `insert` — prevents inserting new blocks, but allows moving or removing existing ones.
+- `false` — prevents locking from being applied to an `InnerBlocks` area even if a parent block contains locking.
```
Comment (gziolo) [inline editor/components/inner-blocks/README.md]: Not related to this PR: How do you feel about converting `false` to `none` string so we could use only one variable type?

File: editor/components/inner-blocks/README.md [hunk 2]
```diff
@@ -38,11 +38,72 @@ registerBlockType( 'my-plugin/my-block', {
...
+
+If locking is not set in an `InnerBlocks` area: the locking of the parent `InnerBlocks` area is used.
+
+If the block is a top level block: the locking of the Custom Post Type is used.
+* **Type:** `Array<String>`
```
Comment (gziolo) [inline editor/components/inner-blocks/README.md]: Should we remove this line, too?

**Review by gziolo — APPROVED** [review]
I added two small corrections where I noticed obsolete lines.

File: editor/components/inner-blocks/README.md (hunk 2 above)
Comment (gziolo) [reply editor/components/inner-blocks/README.md]: I removed, it didn't fit here :)

**Comment by albedo0** [comment]
Hello,

Where can i find a list of the arguments for the blocks like "placeholder, align,..."

**Comment by tofumatt** [comment]
Hi @albedo0,

In the future please open a new issue or ask a question in the `#core-editor` channel in the [Make WordPress Slack](http://make.wordpress.org/chat/).

If you're looking for the props each block accepts, right now the best option, I think, is to look at the code for each block as I don't think we have a set of API docs for the core blocks.

If you'd like to ask something else or follow up please do it outside this pull request as it's unrelated and this is closed 😄

**Pull request merged**